lib/filestream/filestream.go

```diff
@@ -334,6 +334,6 @@ var bwPool sync.Pool
 
 type streamTracker struct {
 	fd     uintptr
-	offset uint64
-	length uint64
+	offset uint64 // nolint
```

Comment:
It is better to revert changes not related to the changes in app/vmselect/promql package.